_layouts/default.html

```diff
@@ -12,6 +12,8 @@
     {% include open-graph.html %}
     <link rel="stylesheet" href="{{ site.cdnurl }}{{ site.baseurl }}/css/pygments-github.css">
     <link rel="stylesheet" href="{{ site.baseurl }}/css/vendors.css">
+    <link rel="stylesheet" href="{{ site.cdnurl }}{{ site.baseurl }}/synapse/dist/scoped-synapse.css">
+    <link rel="stylesheet" href="{{ site.cdnurl }}{{ site.baseurl }}/css/devcenter.css">
```

Comment:
We should update these lines to remove `{{ site.cdnurl }}` as they're currently 404ing.

Reply:
They are? Are you running the dev config?

Reply:
Okay, I do see the fonts getting blocked. Updating!